### PR TITLE
Cassiopeia Privacy Consent - Display Options Inline

### PR DESCRIPTION
**Pull request by coolcat-creations**
Pull Request for Issue #37645  replaces PR #37708

### Summary of Changes
The Privacy Consent decision is placed inline. It saves space and has a better usability 

### Testing Instructions
Turn on the Privacy consent plugin 
Turn on the Terms of Service plugin
Apply Patch an see if the radio buttons on the registration page and on the profile edit page display correct

### Actual result BEFORE applying this Pull Request
grafik

### Expected result AFTER applying this Pull Request
grafik

### Documentation Changes Required
No

### CODE REVIEW

File: build/media_source/templates/site/cassiopeia/scss/blocks/_form.scss [hunk 1]
```diff
@@ -132,3 +132,12 @@ fieldset {
 .com-users-login__options {
   margin-top: $cassiopeia-grid-gutter*2;
 }
+
+.com-users-profile__edit, .com-users-registration {
+  #jform_privacyconsent_privacy, #jform_terms_terms {
+    .radio {
```
Comment (Quy) [inline build/media_source/templates/site/cassiopeia/scss/blocks/_form.scss]:
```suggestion
.com-users-profile__edit,
.com-users-registration {
  #jform_privacyconsent_privacy,
  #jform_terms_terms {
    .radio {
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 38f5fc83b927400c65182f5f0691b8d54732fab3<br /><br />with Prebuilt Package<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37709">issues.joomla.org/tracker/joomla-cms/37709</a>.</sub>

**Comment by joomleb** [comment]
Hi guys, 
thinking about seeing this little fixed improvement in the 4.1.3 release, Please, What is the plan here?

**Comment by coolcat-creations** [comment]
We need at least two Tests to possibly merge it into core

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 27e642c2848b27d67db975bf6817617f79b11669<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37709">issues.joomla.org/tracker/joomla-cms/37709</a>.</sub>

4.1.3-rc2-dev+pr37709 Package

**Comment by coolcat-creations** [comment]
Two tests from independent people ;-) 🤗 but thank you @ChristineWk - @joomleb do you want to test as well?

**Comment by joomleb** [comment]
@coolcat-creations 
thanks for your reply.
I tried it on a Joomla 4.1.3 installation [here](https://test1.egulp.net/registration) by override the .../media/templates/site/cassiopeia/scss/blocks/_form.scss changed file (cleared Joomla cache, tested with Chrome and Firefox anonymous browsing), but as far as I see it is not running. 
Please What am I missing?

**Comment by coolcat-creations** [comment]
Did you install a prebuilt package applied here in the PR?

**Comment by joomleb** [comment]
@coolcat-creations 

"...Did you install a prebuilt package..." = No, as I wrote I just tried it on a Joomla 4.1.3 installation [here](https://test1.egulp.net/registration) **by override** the .../media/templates/site/cassiopeia/scss/blocks/_form.scss **changed file**

"...applied here in the PR?" = Please, Where and How ?
- I think I got the link here above
- I downloaded the "joomla-cms-fix-37708b" zip package
- I installed it in the Joomla 4.1.3 installation 
- Nothing is changed (the link I gave you)

**Comment by Quy** [comment]
Please scroll to the bottom of this page. Look for the link to `Show all checks` to see the link to download the package.

**Review by joomleb — APPROVED** [review]
@Quy 
Many Thanks for help, got it, tested it successfully

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37709">issues.joomla.org/tracker/joomla-cms/37709</a>.</sub>

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**